controller/update.go

```diff
@@ -144,6 +144,8 @@ func (c controller) addFirst(ctx context.Context, req Request, opts UpdateOption
 }
 
 func (c controller) runAddWorker(ctx context.Context, req Request, opts UpdateOptions) (Request, error) {
+	c.Config.Logger.Info(nil, "controller: adding units %v", req.SliceIDs)
```

Comment:
@Nesurion argumented the additional contextual information is too much for the info output. I agree with that. So I removed the context. Now there are all information that are needed. In case you want to have more information you can use the debug log level, that is `-v`. I don't think the context is useful here. I even think the context is very much useless because of its limited design, but maybe I simply get something wrong. Anyway, I am open about this. Tell me how it should be.